k8s/secret-template.yaml

```
@@ -0,0 +1,34 @@
+kind: Secret
+metadata:
+  name: ithaka-backend-secret
+  namespace: ithaka-backend
+  labels:
+    app: ithaka-backend
+type: Opaque
+data:
+  # OPENAI_API_KEY debe ser agregado en base64
+  # Para agregar: echo -n "tu-api-key" | base64
+  OPENAI_API_KEY: ""
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)